aeneas/src/os/Linux.v3

```diff
@@ -285,6 +263,31 @@ class ElfSections(elf: ElfHeader) {
 	def finishStringSection(w: DataWriter) {
 		str.sh_size = w.pos - str_start;
 	}
+	// Add symbol entry and write method name to .strtab section
+	def addMethodSymbol(w: MachDataWriter, mach: MachProgram, prog: Program) {
+		addFuncSymbol(w, ".entry", mach.entryStub);
+		if (mach.allocStub != null) addFuncSymbol(w, ".alloc", mach.allocStub);
+		if (mach.signalStub != null) addFuncSymbol(w, ".signal", mach.signalStub);
+
+		var buf = StringBuilder.new();
+		for (i < prog.ir.methods.length) {
```

Comment:
Oh, I meant just to pull out the body of the loop that works on a single method.

Reply:
What about the stringBuilder? or use `addFuncSymbol()` as well?

Reply:
Yeah, you can pass in the StringBuilder as a temporary. I didn't quite understand the `addFuncSymbol()` comment. If you go back to before extracting this method and only extract the loop body, I think that solves that.